data/json/npcs/exodii/exodii_merchant_missions.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "id": "RUBIK_ANUS_FETICK",
+    "type": "mission_definition",
+    "name": { "str": "Bring two anesthesia kits to Rubik" },
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 2,
+    "value": 240,
+    "item": "anesthetic_kit",
```

Comment:
Anesthetic kit is the rubber hose+glass jar combo autolearned at healthcare 3, I think sending you for the `anus fetic` itself would make sense (and that makes you jump through a lot more hoops if you want to craft it).

Reply:
Forgot this was exclusive of charges, yeah I'll have to change that to specific drugs